Small fixes, changes, etc.

respiten opened the pull request:
## About the Pull Request

- re-routes body disposal in the CDZ to the Re-Education Departures, replaces the 'body disposal' with a a food/snack vending machine and a table with donuts for guards to pass through.
- removes two random closets in the D-Class Shower areas
- adds three possible random contraband locations in mining ruins and adds some random tool placements in the mining area and mining ruins.
- removed ugly in-wall AED cabinets and replaced them with nanomed cabinets in the ULCZ
- expanded on atmosia in the CDZ, giving priority to cryo/spawn locations (yes, I left the shutoff valves where they were so you can still suffocate the whole prison.)
- added a electric stove to EZ (it was EZ, get the pun?)
- adds snack and cola machine to HCZ offices.
- adds a small 3x3 break room across from the offices in the LCZ (where is the water cooler at? how do I talk shit about rebecca if there ain't no damn water cooler)
- changes the windows for the GOI to be tintable, changes GOI access to adminlvl2 and gives goi adminlvl2 access.
- removed CHIEF_ENGINEER access as that isn't being used in our current codebase for the mapping and replaced it with ENGINEERING_LEVEL_5.
- adds logistics equipment to logistics (approved stamp, denied stamp, filing cabinet, paper shredder.) also fixes all approved stamps that were previously invisible.
- I forgot to hit save after making minor adjustments to the spawn location of the stamps.

## Why It's Good For The Game

Ninety nine percent of what was done here was requested in the #mapping or #mapping-suggestions discord channels.

## Changelog

:cl: Respiten
add: fixed approved-stamps being invisible.
add: chief-engineer access.
add: moved body disposal in the CDZ to re-education departures, added vending machines and logistics equipment.
add: added contraband spawn locations and other small improvements.
/:cl:

respiten committed:
small changes

*adds coverage for the CDZ to include important spawn areas and other areas left out.

Re-Routes Body Disposal to Re-Ed Departure

*removes body disposal room from the DCZ, replaces it with a snack, drink vendor and table with a donut box on it.

*reroutes disposal pipes up to the Re-Ed Departure Morgue, deletes two of the morgue units and puts the disposal outlet in the top right corner of the room.
*airlocks for the body disposal are replaced with glass security bubbles one (shouldn't be high security anyways???)

two random closets in D-class showers

removes the two 'Coroner' closets in D-Class showers.

Random Tools and Contraband - small

adds a random contraband item to the north and south mining ruins and a few minor random tools in the ruins and in mining.

removes ugly in-wall AED cabinets and replaces it with NanoMed in the CDC

*replaces two 'in wall' AED 'cabinets' (desk with aed and medkit on top) with Emergency Nanomed's.

adds electric stove to EZ Kitchen

*adds electric stove to EZ Kitchen

adds snack and cola machine to HCZ Offices

adds small breakroom across from LCZ office

*small break room with water cooler, snack and coffee vendor for the office staff.

adds tinted windows and access change to GOI

changes goi door to admin2 and gives tinted windows.

gives GOI Rep access to his own door.

changes CE locker to proper access requirement: ACCESS_ENGINEERING_LEVEL5

ACCESS_ENGINEERING_LEVEL5

adds logistics equipment to logistics, minor fixes

fixes protolathe access

adds approved, denied stamps, paper shredder and file cabinet.

*fixes all approved stamps (icon state is missing, changed it to the boss one which is the same as the approved one.)

fixes logistic stamp placement in logistics.

fixing failed test, the illusive S

fixing the second S of doom.

bug fixes, thank you tests

I wish somebody told me

Window switch moved because of the proximity to the o5's office causing issues.

removes invisible walls in south mining ruins.

for obvious reasons.

Update site53-2.dmm

hry-gh reviewed (changes requested):
map tester thinks you've got a stray cable, look at the check and fix, please!

hry-gh reviewed (changes requested):
merge conflicts!! terrifying

respiten committed:
small changes

*adds coverage for the CDZ to include important spawn areas and other areas left out.

Re-Routes Body Disposal to Re-Ed Departure

*removes body disposal room from the DCZ, replaces it with a snack, drink vendor and table with a donut box on it.

*reroutes disposal pipes up to the Re-Ed Departure Morgue, deletes two of the morgue units and puts the disposal outlet in the top right corner of the room.
*airlocks for the body disposal are replaced with glass security bubbles one (shouldn't be high security anyways???)

two random closets in D-class showers

removes the two 'Coroner' closets in D-Class showers.

Random Tools and Contraband - small

adds a random contraband item to the north and south mining ruins and a few minor random tools in the ruins and in mining.

removes ugly in-wall AED cabinets and replaces it with NanoMed in the CDC

*replaces two 'in wall' AED 'cabinets' (desk with aed and medkit on top) with Emergency Nanomed's.

adds electric stove to EZ Kitchen

*adds electric stove to EZ Kitchen

adds snack and cola machine to HCZ Offices

adds small breakroom across from LCZ office

*small break room with water cooler, snack and coffee vendor for the office staff.

adds tinted windows and access change to GOI

changes goi door to admin2 and gives tinted windows.

gives GOI Rep access to his own door.

changes CE locker to proper access requirement: ACCESS_ENGINEERING_LEVEL5

ACCESS_ENGINEERING_LEVEL5

adds logistics equipment to logistics, minor fixes

fixes protolathe access

adds approved, denied stamps, paper shredder and file cabinet.

*fixes all approved stamps (icon state is missing, changed it to the boss one which is the same as the approved one.)

fixes logistic stamp placement in logistics.

fixing failed test, the illusive S

fixing the second S of doom.

bug fixes, thank you tests

I wish somebody told me

Window switch moved because of the proximity to the o5's office causing issues.

removes invisible walls in south mining ruins.

for obvious reasons.

Update site53-2.dmm

respiten commented:
> merge conflicts!! terrifying

hry-gh commented:
handy for fixing conflicts w/o re-doing everything: https://github.com/ParadiseSS13/Paradise/discussions/16402

respiten commented:
I’ll have it fixed by tonight, thank you for the quick reply Harry!

On Tue, Aug 9, 2022 at 8:17 AM harryob ***@***.***> wrote:

> ***@***.**** requested changes on this pull request.
>
> map tester thinks you've got a stray cable, look at the check and fix,
> please!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Foundation-19/Foundation-19/pull/657#pullrequestreview-1066707570>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A2M4EOLXVGIKUMBMVHO2PITVYJK7BANCNFSM55654QWA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
